src/app/Models/input/input.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="form__group">
+  <label class="form__label">{{ label }}</label>
+  <input
+    class="form__input"
+    [class]="className"
+    [type]="type"
+    [placeholder]="placeholder"
+    [formControl]="control"
+  />
+
+  <ng-content></ng-content>
```

Comment:
why are we using ng-content here if it doesnt have any use case in this PR. please remove it if its not needed.

Reply:
Bhai, from the component page, inside the selectors I'm sending error message and to show this I used ng-content, and finally the whole things are wrapped inside the form-group class. this approach helps me to isolate the individual group separately. Is my approach is right or wrong?

Reply:
Ahh, in that case you can keep it bhai. Thank you.